widgetastic_manageiq.py

```diff
@@ -1108,3 +1109,27 @@ def select(self, title):
     @property
     def selected(self):
         return next(btn.title for btn in self._view_buttons if btn.active)
+
+
+class Search(View):
+    """ Represents search_text control
+    # TODO Add advanced search
+    """
+    search_text = Input(name="search_text")
+    search_btn = Text("//div[@id='searchbox']//div[contains(@class, 'form-group')]"
```

Comment:
@mfalesni I thought these locators are too specific to be used else where so made them this way